aten/src/ATen/core/dispatch/Dispatcher.cpp

```diff
@@ -53,14 +53,27 @@ c10::optional<OperatorHandle> Dispatcher::findSchema(const OperatorName& overloa
   });
 }
 
+c10::optional<OperatorHandle> Dispatcher::findSchema(const OperatorName& overload_name) {
```

Comment:
I think `findSchema` is a misleading name. Maybe `findFullyDefinedOperatorByName`?

Reply:
> I think `findSchema` is a misleading name. Maybe `findFullyDefinedOperatorByName`?

Please let's avoid these super long names 🙏 😁 

First off - personally I'd vote for letting this land without further renames, unless Ed has the appetite to do more fiddling, or we're really worried about churn in the gap between this landing and a cleanup pass. 

Buuuut whenever we're ready for name changes, I'd really emphasize how much of a readability burden these sentence-in-disguise names are - the extra signal is less than meets the eye, and doesn't compensate for the added bulk. 

E.g. here's my pitch for how `findOperatorByName` and `findFullyDefinedOperatorByName` can be shortened to `findOp` and `findDefinedOp` without loss of signal, and at great benefit to readability:
- `Fully` isn't necessary, since the "defined" condition is binary
- `Operator` can be shortened, since "op" is used idiomatically everywhere and doesn't risk being mistaken for anything else
- `ByName` isn't necessary for either method name, since operator names as lookup keys is a ubiquitous and exclusive concept in the API.

Alt: `findOp` and `findOpDef`. 

For: fully canonicalizes the API vocabulary work "def" to mean an op in defined state, plus it's even shorter. 

Against: at a quick glance, the subset/superset relationship between the two  is less clear (i.e. "op" is all ops, "defined op" is the subset with the property of being defined). Worst case they just read, confusingly, as synonyms (i.e., takes a little reading to answer the question, aren't all ops op defs?).

Reply:
> First off - personally I'd vote for letting this land without further renames, unless Ed has the appetite to do more fiddling, or we're really worried about churn in the gap between this landing and a cleanup pass.

I specifically don't want to rename `findSchema` in this patchset, only because it's a semi-widely used API and I'd have to fix a bunch of call sites upon renaming it. I can rename `findOperatorByName` to `findOp` immediately. Although, I don't mind as much using the long form name here.

Reply:
Renamed in https://github.com/pytorch/pytorch/pull/35061/commits/e094ff0c3cd92498c574d2bd14017cb32b9a8862